ntbs-service/Models/Entities/ClinicalDetails.cs

```diff
@@ -81,6 +81,7 @@ public partial class ClinicalDetails : ModelBase, IOwnedEntityForAuditing
 
         [Display(Name = "Notes")]
         [MaxLength(1000, ErrorMessage = ValidationMessages.MaximumTextLength)]
+        [ContainsNoTabs]
         [RegularExpression(ValidationRegexes.CharacterValidationWithNumbersForwardSlashExtendedWithNewLine,
             ErrorMessage = ValidationMessages.InvalidCharacter)]
```

Comment:
I had hoped to be able to add a two regex attributes, but sadly that wasn't allowed so I needed to make a custom one `ContainsNoTabs`, not that that was very hard